### PR TITLE
fix: unused composables/use-browser-detection

### LINKED ISSUE #2535: Remove unused `composables/use-browser-detection`
## Problem

`composables/use-browser-detection` is unused but still exists in our code.

## Description

Remove the `composables/use-browser-detection` file and the useless Jest mock in `v-audio-track.spec.js` against it.

### PR DESCRIPTION
**Pull request by AlimurtuzaCodes**
## Fixes
Fixes #2535  by @sarayourfriend

## Description
Removed unused `composables/use-browser-detection` file and also removed useless Jest mock in `v-audio-track.spec.js` against it.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/test/unit/specs/components/AudioTrack/v-audio-track.spec.js [hunk 1]
```diff
@@ -19,9 +19,6 @@ jest.mock("~/composables/use-match-routes", () => ({
   useMatchSingleResultRoutes: jest.fn(() => ({ matches: { value: false } })),
 }))
 
-jest.mock("~/composables/use-browser-detection", () => ({
-  useBrowserIsBlink: jest.fn(() => false),
-}))
```
Comment (dhruvkb) [inline frontend/test/unit/specs/components/AudioTrack/v-audio-track.spec.js:22]: Please also remove this blank line. That should make the lint check pass.

**Review by sarayourfriend — APPROVED** [review]
LGTM! Thank you @AlimurtuzaCodes :pray:

File: frontend/test/unit/specs/components/AudioTrack/v-audio-track.spec.js (hunk 1 above)
Comment (AlimurtuzaCodes) [reply frontend/test/unit/specs/components/AudioTrack/v-audio-track.spec.js]: Ok Done.

**Review by obulat — CHANGES_REQUESTED** [review]
Actually, `$ua` is used in the `ui` store and other stores. We should update them, too. Could you please run a search for `$ua` in the frontend directory, and remove it. 

The `isMobileUa` property needs to be removed from the `ui` store:
https://github.com/WordPress/openverse/blob/6059a03c06fc8d6e3dcee93d06f2616fd8e8cddc/frontend/src/stores/ui.ts#L43

**Review by obulat — APPROVED** [review]
Thank you for your contribution, @AlimurtuzaCodes ✨

**Pull request merged**